mitmproxy/web/app.py

```diff
@@ -26,7 +25,7 @@ def _strip_content(flow_state):
                 continue
             if message["content"]:
                 message["contentLength"] = len(message["content"])
-            elif message["content"] == CONTENT_MISSING:
+            elif message["content"] is None:
```

Comment:
This `if elif else` can be simplified to `if else` by combining the cases where content is not None.